### PR TITLE
Added paragraph about adding an ALR badge

### PR DESCRIPTION
**Pull request by Jellix**
I just saw that neat little thing that you can easily link your project back to Alire, so I added a paragraph on how to do that.

Could be a nice little gimmick to improve visibility.

### CODE REVIEW

File: doc/publishing.md [hunk 1]
```diff
@@ -209,3 +209,31 @@ link, you can follow the usual procedure to submit a PR to a github repository:
...
 available for normal use. The open source Ada ecosystem needs all the help it
 can get, so thank you for contributing!
+
+## ALR Badge
+
+If you like, you can even add a nice, shiny badge to your project page to link
+back to the [Alire website](https://alire.ada.dev). This can even serve as a
+reminder to republish your project once you created a new release, because
+the badge shows the latest version of your project that is known to Alire.
+It might take a while for the global Alire index to get updated after we
```
Comment (Fabien-Chouteau) [inline doc/publishing.md]: Actually the website is updated every day, once per day.

File: doc/publishing.md [hunk 2]
```diff
@@ -209,3 +209,31 @@ link, you can follow the usual procedure to submit a PR to a github repository:
...
+accepted your pull request. Just be a bit patient, please.
+
+To add the badge, all you need to do is add the line
+
+```
+[![Alire](https://img.shields.io/endpoint?url=https://alire.ada.dev/badges/YOUR_PROJECT.json)](https://alire.ada.dev/crates/YOUR_PROJECT.html)
+```
+
+to your `README.md`. Of course, you need to replace the string `YOUR_PROJECT` with
```
Comment (Fabien-Chouteau) [inline doc/publishing.md]: We use the word `crate` instead of `project` to avoid confusion with gprbuild project and project file.

**Review by Fabien-Chouteau — CHANGES_REQUESTED** [review]
Good idea @Jellix !

I have a couple of comments.

File: doc/publishing.md (hunk 1 above)
Comment (Jellix) [reply doc/publishing.md]: Clarified that the change may take up to a day.

File: doc/publishing.md (hunk 2 above)
Comment (Jellix) [reply doc/publishing.md]: Good catch. I am still getting used to it. :)

**Review by Fabien-Chouteau — APPROVED** [review]

**Comment by mosteo** [comment]
Neat! I was unaware of these json files in the website...

**Pull request merged**